src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -186,6 +205,23 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
+            if (commandResult.isTaskRelated()) {
+                outputPanel.handleTask(logic.getFilteredPersonList().get(0));
+            }
```

Comment:
I think instead of using `logic#getFilteredPersonList#get`, you can standardize across other types of command results and use `logic#getPatientOfInterest`. You can update the rest of the task related commands accordingly by adding an extra `model#setPatientOfInterest` just like what you did for the patient related commands.

Reply:
Good idea, updated